src/Sanitisers/ApplicantSanitiser.php

```diff
@@ -102,4 +102,48 @@ public static function sanitiseAdditionalFields(array $applicant): array
 
         return $applicant;
     }
+
+    /**
+     * Sanitise edaid for editing student profile
+     *
+     * @return ?int sanitised $savedProfileFieldEdAid
+     */
+    public static function sanitiseEdAid($savedProfileFieldEdAid): ?int
+    {
+        return $savedProfileFieldEdAid ? (int)$savedProfileFieldEdAid : null;
```

Comment:
this for all your functions:
it's weird that you're checking whether the field is there inside the sanitising function. it would be better to only attempt to sanitise the field if it's there. otherwise why would you pass an empty value to a function, then figure out that it's empty and decide to do nothing? you could have checked before calling the function.

Reply:
as a result of that comment, all that would remain in this function is a typecast to int: (int) $savedProfileFieldEdAid. so then it is questionable whether this function should exist at all